numba/np/arrayobj.py

```diff
@@ -1563,6 +1563,16 @@ def array_T(context, builder, typ, value):
     return impl_ret_borrowed(context, builder, typ, res)
 
 
+@overload(np.logspace)
+def numpy_logspace(start, stop, num=50, base=10.0):
```

Comment:
I don't think this will replicate the NumPy behaviour in its present form. The `endpoint` kwarg is missing and should be located between `num` and `base`. The impact of this is that if a user calls the Numba `np.logspace` implementation without using kwargs they'll get unexpected behaviour. e.g.
```
logspace(5,  10, 50, 10.0) # 10.0 should be `endpoint`!
```
reference docs: https://numpy.org/doc/stable/reference/generated/numpy.logspace.html

Reply:
I have changed logspace implementation to 3-argument form to avoid confusion between `endpoint` and `base` because `endpoint` is not supported now, i  will add full support for `linspace` and `logspace` in subsequent PR.